cmake/toolchains/gnu4-toss_3_x86_64_ib-librom-dev.cmake

```diff
@@ -0,0 +1,22 @@
+# Toolchain for Lawrence Livermore National Laboratory TOSS3 machines
+# (e.g., Quartz), assuming the following dependencies:
+#
+# - MVAPICH2/2.3 with Intel 19.0.4 compilers
+# - Intel MKL
+# - HDF5 1.8.18
+
+# Use MPI compiler wrappers because it simplifies detection of MPI
+set(CMAKE_C_COMPILER mpicc)
+set(CMAKE_CXX_COMPILER mpicxx)
+set(CMAKE_Fortran_COMPILER mpif90)
+set(BLA_VENDOR Intel10_64lp)
+set(HDF5_ROOT /usr/tce/packages/hdf5/hdf5-serial-1.8.17-gcc4.9.3)
```

Comment:
@kevinhkhuynh @chldkdtn I am unhappy with the HDF5_ROOT line in this toolchain. I set the compilers to what is currently loaded, but the HDF5_ROOT is hard-coded to the one compiled by gcc4.9.3, which is okay for me since that's the compiler that's loaded in my environment.

Is there a way to automate this to the HDF5 library currently loaded?

For example, the bash equivalent would be something like:
```
H5DIFF_SUFFIX=/bin/h5diff
H5DIFF_PATH=$(which h5diff)
export HDF5_ROOT=${H5DIFF_PATH%$H5DIFF_SUFFIX}

```

Reply:
Resolved!